client/verta/tests/unit_tests/registry/test_check_model_dependencies.py

```diff
@@ -0,0 +1,117 @@
+# -*- coding: utf-8 -*-
+""" Unit tests for verta.registry.check_model_dependencies """
+
+import pytest
+from typing import Set
+import warnings
+
+from verta.registry.check_model_dependencies import check_model_dependencies
+from verta.environment import Python
+
+
+# TODO VRD-682 convert module names to pkg names and update this
+@pytest.fixture(scope='session')
+def complete_env() -> Python:
+    """ Environment with all expected packages from dependency_testing_model
+    fixture.  For completeness, some are versioned and some are not.
+    """
+    return Python([
+        'calendar',
+        'click',
```

Comment:
I'm not sure if this was intentional, but it looks like you're exploiting a fallback in `Python` 🤔 

So, `Python()` is supposed to insta-magically identify and assign version numbers to all libraries. For example,

```python
env = Python(["click"])
assert env.requirements == ["click==8.1.3", "cloudpickle==2.2.0", "verta==0.22.0"]
```

but it will skip this behavior if it encounters an error while checking the environment for version numbers. In this case, because `calendar` comes from the std lib and doesn't have a `__version__` attribute, `Python()` encounters an error and just rolls with `["calendar", "click", ...]`. If you set your logging level to `INFO`, you'll see

```
INFO - failed to manually parse requirements; falling back to capturing raw contents
Traceback (most recent call last):
  File "/Users/miliu/Documents/modeldb/client/verta/verta/environment/_python.py", line 266, in _capture_requirements
    _pip_requirements_utils.set_version_pins(requirements_copy)
  File "/Users/miliu/Documents/modeldb/client/verta/verta/_internal_utils/_pip_requirements_utils.py", line 267, in set_version_pins
    six.raise_from(error, None)
  File "<string>", line 3, in raise_from
ValueError: unable to determine a version number for requirement 'calendar'; it might not be installed; please manually specify it as 'calendar==x.y.z'
```

Reply:
All that is to say: If you're intentionally doing this so that `complete_env` stores libraries without version strings, I'd recommend adding a comment

```suggestion
        'calendar',  # from std lib; forces Python() to skip capturing version number
        'click',
```

Reply:
I just gave all package strings proper versions for now under the assumption we'll handle all this package related stuff all at once in the next block of work. (VRD-682)

Reply:
I don't think VRD-682 has anything to do with whether `Python()` captures library version numbers. Using my suggestion of adding a comment really would've been fine.